samples/CosmosDBIntegration/CosmosDbIntegrationTutorial.md

```diff
@@ -0,0 +1,117 @@
+1. Direct to your local.settings.json file in Visual Studio.
+2. Copy and paste the following code as a replacement for your local.settings.json.
+3. Replace each variable with the appropriate value.
+ ```json
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "CosmosDbDatabaseId": "<database-id>",
+    "ListCosmosDbContainerId": "<container-id-for-list>",
+    "PubSubCosmosDbContainerId": "<container-id-for-pubsub>",
+    "MessagesCosmosDbContainerId": "<container-id-for-pubsub-messages>",
+    "PubSubChannel": "<pubsub-channel-name>",
+    "StreamCosmosDbContainerId": "<container-id-for-stream>",
+    "StreamCosmosDbContainerIdSingleDocument": "<container-id-for-stream-singular-document>",
+    "StreamTest": "<stream-name>",
+    "StreamTestSingleDocument": "<stream-name-singular-document>",
+    "RedisConnectionString": "<cache-name>.redis.cache.windows.net:6380,password=<access-key>,ssl=True,abortConnect=False,tiebreaker=",
+    "CosmosDbConnectionString": "AccountEndpoint=https://<cosmosdb-account>.documents.azure.com:443/;AccountKey=<access-key>;"
+  }
+}
+ ```
+ NOTE: each trigger should be using a separate container, which is dependent on the variable names.
+
+ #### Description of each variable
+ * CosmosDbDatabaseId: Database all of the functions can share
+ * ListCosmosDbContainerId: Container for Lists
+ * PubSubCosmosDbContainerId: Container for key/value pairs set with PubSub triggers
+ * MessagesCosmosDbContainerId: Container for messages sent with PubSub triggers
+ * PubSubChannel: The name of the channel the PubSub trigger is listening to
+ * StreamCosmosDbContainerId: Container IDs of the CosmosDB containers that store the stream messages
+ * StreamCosmosDbContainerIdSingleDocument: Container IDs of the CosmosDB containers that store the stream messages when writing to a single document
+ * StreamTest: Name of the streams the trigger is listening to
+ * StreamTestSingleDocument: Name of the stream the trigger is listening when writing to a single document in CosmosDB
+ * RedisConnectionString: Primary connection string to Redis Cache
+ * CosmosDbConnectionString: Account endpoint to CosmosDB
+
+### Adjusting List and Stream variables
+Lists 
+1.	WriteBehind
+    - Change the value of ListKey to the desired key before the function 
+2.	WriteAround 
+    - Change the value of ListKey to the desired key before the function 
+3. ReadThrough
+    - Change the value of ListKey to the desired key before the function
+    ```csharp
+    public const string ListKey = "userListName";
+    ```
+    
+
+### Security
+* This design introduces new secrets. Theses secrets are the CosmosDB keys and connection strings, and the Azure Cache for Redis Connection strings. These will be stored in the customers’ storage account associated with their function app.
+
+## Clean up resources
+
+If you're not going to continue to use this application, delete
```

Comment:
We have boilerplate language for this that would be better but this is a good thing to add. Not a requirement but we generally like having this because it puts we want to remind customers that they sometimes are creating resources that they might forget until the bill comes due.